tests/model_forms/tests.py

```diff
@@ -357,6 +357,21 @@ class Meta:
         self.assertEqual(list(ExcludeFields.base_fields),
                          ['name', 'slug'])
 
+    def test_exclude_overridden_fields(self):
+        class NoExcludeFields(forms.ModelForm):
+            url = forms.URLField()
+
+            class Meta:
+                model = Category
+                exclude = []
+
+        class ExcludeFields(NoExcludeFields):
+            class Meta:
+                model = Category
+                exclude = ['url']
```

Comment:
Let me know if you'd rather have `type()` calls, I thought it'd be a bit more readable this way.